fastlane/Fastfile

```diff
@@ -777,10 +777,10 @@ platform :ios do
     build_for_prototype_build
 
     release_notes = <<~NOTES
-      Pull Request: ##{pull_request_number || 'N/A'}
       Branch: `#{ENV.fetch('BUILDKITE_BRANCH', 'N/A')}`
       Commit: #{ENV.fetch('BUILDKITE_COMMIT', 'N/A')[0...7]}
     NOTES
+    release_notes += "Pull Request: ##{PULL_REQUEST_NUMBER}" unless PULL_REQUEST_NUMBER.nil?
```

Comment:
Let me know what you think, but if we have builds from `trunk` which definitely won't have a PR, it would seem noisy to read "Pull Request: N/A" in the release notes, especially at the top.